pydata_center/monitoring/tests.py

```diff
@@ -1,3 +1,52 @@
-from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
 
-# Create your tests here.
+from .models import ServerStatus
+
+
+class ReceiveStatusEndpointTests(APITestCase):
+    def setUp(self):
+        self.url = reverse('monitoring:receive_status')
```

Comment:
If you're not modifying data between tests, `setUpTestData` is more efficient as it's only run once per class.